README.md

```diff
@@ -15,25 +37,26 @@ dotnet build --configuration Release
 To run the tests, use this command:
 
 ```
-dotnet test --settings Casper.Network.SDK.Test/test.runsettings
+dotnet test --filter "TestCategory!~NCTL"
```

Comment:
@davidatwhiletrue the command above fails becuse of double quotes on Windows:
```
$ dotnet test --filter "TestCategory!~NCTL"
zsh: event not found: ~NCTL
```

Can we changes it to single quotes? Do they work on Windows?

Reply:
Changed to single quotes.